### PR TITLE
Release v3.0.0-alpha.6

### PR DESCRIPTION
**Pull request by james-nash**
Version bumps from release ` v3.0.0-alpha.6` NPM package.

### CODE REVIEW

File: packages/gravity-ui-nunjucks/package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
   "devDependencies": {
     "@buildit/eslint-config": "^1.0.0",
     "@buildit/gravity-particles": "^0.3.0",
-    "@buildit/gravity-ui-web": "^3.0.0-alpha.2",
+    "@buildit/gravity-ui-web": "^3.0.0-alpha.6",
```
Comment (dw-buildit) [inline packages/gravity-ui-nunjucks/package.json]: Is there not an equivalent `package-lock.json` change for the nunjucks package?

Comment (james-nash) [reply packages/gravity-ui-nunjucks/package.json]: Nope.

When Lerna does it's "bootstrap" thing, it temporarily removes any local packages dependencies, runs the install and then restores the `package.json`. So, the `package-lock.json` files don't contain refrences to any of the other packages in this monorepo

Comment (james-nash) [reply packages/gravity-ui-nunjucks/package.json]: I suspect I didn't really need to update the Nunjucks `package.json` at all, but it felt neater to keep it up-to-date.

Especially later, when we start publishing that to NPM, it'll be important for it's `package.json` to list the correct version of `gravity-ui-web`

Comment (dw-buildit) [reply packages/gravity-ui-nunjucks/package.json]: Makes sense, i saw the `gravity-ui-web` lock being updated but of course that's the actual package's version.

**Pull request merged**